src/main/java/com/offdk/play/persistence/PlayerRepository.java

```diff
@@ -7,5 +7,5 @@
 import com.offdk.play.model.game.Player;
 
 public interface PlayerRepository extends CrudRepository<Player, Long> {
-  Optional<Player> findOneByUserUserId(String userId);
```

Comment:
yeah it is fixed, but that's how repository works.  The field used to be user.getUerId().  Now it is user.getId().